Functions.Templates/Templates/BlobTrigger-TypeScript/index.ts

```diff
@@ -0,0 +1,7 @@
+import { AzureFunction, Context } from "@azure/functions"
+
+const blobTrigger: AzureFunction = async function (context: Context, myBlob) {

```

Comment:
Any reason myBlob isn't type "any"? Same comment applies for all templates without types in parameters